content/minfraud/api-documentation/_schemas/RequestDevice.mdx

```diff
@@ -17,6 +17,9 @@ import requestJson from '../_examples/request';
     The IP address associated with the device used by the customer in the
     transaction. The IP address must be in IPv4 or IPv6 presentation format,
     i.e., dotted-quad notation or the IPv6 hexadecimal-colon notation.
+
+    [Get tips for how to pass the `/device/ip_address` input on our Knowledge
```

Comment:
Any particular reason changing from Learn more to Get tips? :)

Reply:
in this case, i assume that most customers understand what an IP address is and that they want to pass us the IP address of the end user. the KB article i link to here is more about troubleshooting tips to make sure you aren't sending too many queries, and to make sure you're passing the end user's IP in the event of a CDN or similar